src/loadgenerator/loadgen.sh

```diff
@@ -25,11 +25,11 @@ fi
 set -x
 
 # if one request to the frontend fails, then exit
-STATUSCODE=$(curl --silent --output /dev/stderr --write-out "%{http_code}" http://${FRONTEND_ADDR})
+STATUSCODE=$(curl --silent --output /dev/stderr --write-out "%{http_code}" ${FRONTEND_ADDR})
```

Comment:
Same here - could you please give more details on your use case? Also why can't a user just add `http://<my-custom-addr>:<port>` to the loadgen deployment manifest, as `FRONTEND_ADDR`? why is an extra http needed here? in this PR as is, wouldn't the curl command actually have two `http://http://{FRONTEND_ADDR}`? because it's here, and in the deployment manifest?